README.md

```diff
@@ -33,6 +33,9 @@ Alma.configure do |config|
 
```

Comment:
noting small typo here - missing closing quotation mark. not a blocker, just nice to fix